crates/driver/src/domain/competition/solution/score_calculator.rs

```diff
@@ -0,0 +1,24 @@
+use crate::{boundary, domain::eth, infra::mempool};
+
+#[derive(Debug, Clone)]
+pub struct ScoreCalculator {
```

Comment:
The `ScoreCalculator` doesn't fit as a domain object IMO. A `SolverScore` with a `fn calculate` method makes more sense to me.

Reply:
Renamed. Does DDD suggests some specific naming?